framework/dotnet/BenchPress.TestFramework/MOCK_DELETE_LATER.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections.Generic;
+
+namespace BenchPress.TestEngine
+{
+  namespace Azure
+  {
+    public class Config
+    {
+      public void Set(Config config)
+      {
+      }
+    }
+
+    public class ResourceGroup
+    {
+      public object GetResourceGroup(string nameOrResourceId)
+      {
+        if (nameOrResourceId.Contains("/"))
+        {
+          return GetResourceGroupById(nameOrResourceId);
+        }
+        return GetResourceGroupByName(nameOrResourceId);
+      }
+
+      private object GetResourceGroupByName(string name)
+      {
+        return null;
+      }
+
+      private object GetResourceGroupById(string id)
+      {
+        return null;
+      }
+
+      public bool CheckResourceGroupExists(string nameOrResourceId)
+      {
+        return GetResourceGroup(nameOrResourceId) != null;
+      }
+    }
+
+    public class StorageAccount
+    {
+      public object GetStorageAccount(string nameOrResourceId)
+      {
+        if (nameOrResourceId.Contains("/"))
+        {
+          return GetStorageAccountById(nameOrResourceId);
+        }
+        return GetStorageAccountByName(nameOrResourceId);
+      }
+
+      private object GetStorageAccountByName(string name)
+      {
+        return null;
+      }
+
+      private object GetStorageAccountById(string id)
+      {
+        return null;
+      }
+
+      public bool CheckStorageAccountExists(string nameOrResourceId)
+      {
+        return GetStorageAccount(nameOrResourceId) != null;
+      }
+    }
+  }
+  namespace Kubernetes
```

Comment:
`k8s` is out of scope for the Game Plan

Reply:
yes, I just want to make sure that this namespace/class layout is future-proof and we can use it with Kubernetes, ssh, git, etc. later on.

Reply:
There is no reason why the namespaces wouldnt be future proof. I prefer to not have "dead" code in the codebase so I would agree we should remove this